zero/zero_mq/queue_device/client.py

```diff
@@ -10,6 +10,7 @@
 
 class ZeroMQClient:
     def __init__(self, default_timeout):
+        self._address = None
```

Comment:
Why do we need address here? 🤔

Reply:
pylint complains about it being not set in the initializer - and so far pylint is right, because it's in general good to assign the intended attributes of an instance as `None` (or any other placeholder) until they have a meaningful value, so that 1. the future code reader can see all instance attributes gathered in one place, 2. any time after the instance initialization is done, straightforward accessing these attributes is guaranteed (before `_ensure_connected()` for example) not to raise an AttributeError. This might be useful if you want to extend this code in the feature and maybe add some broker between both mentioned methods. But that's just hypothesizing about why something useful _might_ be particularly really useful.

Reply:
I agree, then there are some "nullable" values we might need to handle.

Reply:
See also https://github.com/Ananto30/zero/pull/34/commits/4acb077dad98a1b200199dd2e0a0eaa2b190eb3d according to https://github.com/Ananto30/zero/pull/34#discussion_r1160604752.